pkg/values/value.go

```diff
@@ -251,7 +265,15 @@ func unwrapTo[T any](underlying T, to any) error {
 		}
 		*tb = underlying
 	default:
-		return fmt.Errorf("cannot unwrap to value of type: %T", to)
+		// Don't break for custom types that are the same underlying type
+		// eg: type FeedId string allows verification of FeedId's shape while unmarshalling
```

Comment:
nit add quotes to the embedded code example:
`type FeedId string`

Reply:
Can you suggest it, then I won't need re-approval from Bolek if I accept the change :D